### PR TITLE
Ship 1.1.0rc1

### PR DESCRIPTION
**Pull request by aanand**

### CODE REVIEW

**Comment by aanand** [comment]
Rebased on master.

File: CHANGES.md [hunk 1]
```diff
@@ -1,6 +1,35 @@
...
+
+Besides that, there’s a lot of new stuff in this release:
+
+- We’ve made a few small changes to ensure that Compose will work with Swarm, Docker’s new clustering tool (https://github.com/docker/swarm). Eventually you'll be able to point Compose at a Swarm cluster instead of a standalone Docker host and it’ll run your containers on the cluster with no extra work from you. As Swarm is still developing, integration is rough and lots of Compose features don't work yet.
+
+- `docker-compose run` now has a `--service-ports` flag for exposing ports on the given service. This is useful for e.g. running your webapp with an interactive debugger.
+
+- You can now link to containers outside your app with the `external_links` option in docker-compose.yml.
+
+- `docker-compose up` now makes fewer API calls and is much faster when running against a Docker host with lots of containers.
```
Comment (dnephin) [inline CHANGES.md]: with the flag `--no-build`, right ?

**Comment by dnephin** [comment]
LGTM

File: CHANGES.md (hunk 1 above)
Comment (aanand) [inline CHANGES.md]: Amended.

**Comment by thaJeztah** [comment]
image

**Comment by marksteve** [comment]
:tada: This made my day! Hurray for `--env-file`!

**Comment by nathanleclaire** [comment]
Hurrah!

**Pull request merged**